src/steps/then.js

```diff
@@ -35,5 +36,21 @@ module.exports = function () {
             }
         });
 
+        Then(_r(`^${pageObject} has text ${dictionaryObject}$`), async function async (element, textPhrase) {
+            /**
+            * @param {pageObject} element - String or "page"."object" to select the element
+            * @type {String} or {DictionaryObject}
+             */
+            const phrase = getDictionaryObject.call(this, textPhrase);
+            const locator = getPageObject(element);
+            const locatorText = browser.getText(locator);
```

Comment:
`browser.getText(locator);` is async tep so it should be marked as await

Reply:
Done